client/functions.lua

```diff
@@ -372,6 +372,29 @@ function QBCore.Functions.SpawnVehicle(model, cb, coords, isnetworked, teleportI
     if cb then cb(veh) end
 end
 
+function QBCore.Functions.SpawnNetworkedVehicle(model, cb, coords, heading, teleportInto)
```

Comment:
Modify the SpawnVehicle function instead

Reply:
I am happy to do this but I believe there are instances where non-networked vehicles are required and vice versa. Thoughts?